package.json

```diff
@@ -59,89 +52,72 @@
     "@openzeppelin/test-helpers": "0.5.5",
     "@truffle/contract": "4.2.4",
     "@truffle/hdwallet-provider": "1.0.34",
-    "@types/big.js": "4.0.5",
-    "@types/bignumber.js": "5.0.0",
     "@types/chai": "4.2.11",
     "@types/chai-as-promised": "^7.1.2",
     "@types/cors": "^2.8.6",
     "@types/eth-sig-util": "2.1.0",
     "@types/express": "^4.17.6",
     "@types/lodash": "^4.14.150",
-    "@types/mocha": "7.0.2",
     "@types/nedb": "1.8.9",
     "@types/node": "13.9.1",
-    "@types/request": "2.48.4",
-    "@types/request-promise": "^4.1.46",
     "@types/sinon": "9.0.0",
     "@types/sinon-chai": "3.2.4",
-    "@types/source-map-support": "0.5.1",
     "abi-decoder": "2.2.2",
     "async-mutex": "^0.2.1",
     "axios": "0.18.1",
-    "big-js": "3.1.3",
+    "bn.js": "5.1.2",
     "big.js": "5.2.2",
+    "body-parser": "1.19.0",
     "chai": "4.2.0",
     "commander": "5.1.0",
-    "esm": "3.2.25",
     "eth-sig-util": "2.5.2",
-    "ethereumjs-abi": "0.6.8",
     "ethereumjs-common": "^1.5.0",
     "ethereumjs-tx": "2.1.2",
     "ethereumjs-util": "6.2.0",
     "ethereumjs-wallet": "0.6.3",
+    "express": "4.17.1",
     "jsonrpc-lite": "2.1.0",
     "lodash": "4.17.13",
+    "minimist": "1.2.5",
     "nedb-async": "0.1.3",
     "ow": "0.17.0",
     "patch-package": "6.2.1",
-    "postinstall-postinstall": "2.0.0",
-    "request-promise": "4.2.2",
     "truffle-hdwallet-provider": "1.0.17",
     "web3": "1.2.6",
     "web3-core": "1.2.6",
     "web3-core-helpers": "1.2.6",
     "web3-eth": "1.2.6",
     "web3-eth-abi": "1.2.6",
     "web3-eth-contract": "1.2.6",
-    "web3-utils": "1.2.6",
-    "webpack-bundle-analyzer": "3.3.2"
+    "web3-utils": "1.2.6"
   },
   "devDependencies": {
-    "@0x/sol-compiler": "4.0.8",
     "@0x/sol-coverage": "4.0.8",
```

Comment:
if remove, the remove all unused 0x packages.

Reply:
these were indicated to be unused by depcheck